### ISSUE TITLE
Separate individuals for scoring

### ISSUE THREAD

**Issue by HelenYSLin** [eldronzhou/SDPR_admix#5]
Hi Geyu,

Thank you for making this useful tool. I was wondering if the tutorial is training and testing on the same set of individuals? 
In the example, the scoring step uses: 
`./score -vcf test/chr22_train.vcf.gz -msp test/chr22_train.msp.tsv -score test/res.txt -out test/test.profile`

If the goal is to apply the weights learned from the training set to an independent test set, should the scoring command instead be:
`./score -vcf test/chr22_test.vcf.gz -msp test/chr22_test.msp.tsv -score test/res.txt -out test/test.profile` ?

Thank you

**Comment by eldronzhou** [comment]
Hi Helen,

Yes, you are correct.
